La'Tonia/binary_search_practice.py

```diff
@@ -0,0 +1,29 @@
+'''
+binary search project to look for certain number in a string
+
+project courtesy https://towardsdatascience.com/10-python-mini-projects-that-everyone-should-build-with-code-769c6f1af0c4
+'''
+
+# code designed to look in middle then the front and then the end of string
+# this exercise introduced me to the double forward slash, which according to article at https://betterprogramming.pub/lesser-known-arithmetic-operators-in-python-a34670087b3a is "known as the integer division operator. Essentially, it will divide the left by the right, and only keep the whole number component." ... n!ce
+# interesting number seven only returns for one position and at location seven when clearly indexed farther along in the list
+lst7 = [1, 3, 0, 2, 44, 5, 6, 67, 9, 8, 37, 99, 10, 64, 13, 88, 7, 56, 7, 21, 70, 91]
+lst7.sort()
+first = 0
+last = len(lst7) - 1
+mid = (first + last) // 2
```

Comment:
Great use of floor division. Had to brush up on it after sharing the ascii code with everyone